### PR TITLE
Enable layout explorer feature

### PR DESCRIPTION
**Pull request by DaveShuckerow**
@albertusangga

### CODE REVIEW

**Comment by kenzieschmoll** [comment]
This change will also need to be reverted for the feature to be enabled in web https://github.com/flutter/devtools/pull/1459/

**Comment by DaveShuckerow** [comment]
> This change will also need to be reverted for the feature to be enabled in web #1459

Thanks, revert commit added.

File: packages/devtools_app/lib/src/inspector/inspector_controller.dart [hunk 1]
```diff
@@ -139,8 +139,7 @@ class InspectorController extends DisposableController
     debugSummaryLayoutEnabled.value = !debugSummaryLayoutEnabled.value;
   }
 
-  // TODO(albertusangga): Remove this flag if required CL to Flutter is landed
-  static bool enableExperimentalStoryOfLayout = false;
+  static bool enableExperimentalStoryOfLayout = true;
```
Comment (albertusangga) [inline packages/devtools_app/lib/src/inspector/inspector_controller.dart]: So sorry for this one, I should have removed it completely, this is unnecessary to enable the layout explorer

**Comment by albertusangga** [comment]
Closing in favor of #1475

**Pull request closed without merging**